frontend/amundsen_application/static/js/config/config-utils.ts

```diff
@@ -344,6 +344,26 @@ export function getDescriptionSourceIconPath(sourceId: string): string {
   return '';
 }
 
+/**
+ * Returns the edit button url for a given description source.
+ * If a configuration does not exist or edit buttons are disabled, empty string
+ * is returned.
+ */
+export function getDescriptionEditUrl(dataSource: TableSource): string | null {
+  const config = AppConfig.resourceConfig[ResourceType.table];
+  const sourceId = dataSource.source_type;
+  if (
+    config &&
+    config.supportedDescriptionSources &&
+    config.supportedDescriptionSources[sourceId] &&
+    config.supportedDescriptionSources[sourceId].renderEditButton
```

Comment:
If I understand well, this would default to false on all the description sources added by users that aren't dremio or github, right?

Reply:
The current behavior would be that the users would need to explicitly set `renderEditButton` for their custom description sources as it's not an optional attribute. I can change it to be optional and if we do that this would default to `undefined`. I intentionally made it required so that the users won't have any unintentional behavior, but I'm open to any suggestions on the whole implementation.

Reply:
I think being backwards compatible outweighs any other consideration at this moment. This is a fairly small change, and we don't want to annoy our users with required changes on our next version just for this change.

Another thing would be if we want to wait to ship this on our next major version, whenever that happens.

WDYT @dorianj ?

Reply:
I am very willing to make the changes for the backwards compatibility, as long as we can reach a consensus.

@dorianj I would be happy to receive your input as well to decide on the approach here. We are trying to solve a relatively small UI issue but I believe a big number of people might potentially have the same issue.

Reply:
Sorry y'all, was on vacation when this happened and missed this.

I don't think this should break back compat... to match current behavior, `undefined` would equate to `true`, which is a bit unintuitive but I think that's OK

Reply:
Thank you @dorianj for checking this. I've made the suggested change so it would be great if you can take a look again. also I've signed-off my last commit but DCO task is still failing for some reason.. if you can let me know how to fix it I can do it

Reply:
I've fixed the checks and implemented the changes, a review would be much appreciated
@dorianj @Golodhros